### PR TITLE
Added text file with some text

### PR DESCRIPTION
**Pull request by christopherc1331**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,284 @@
...
+			width: 100%;
+			padding: 2%;
+			margin-right: 0px;
+			border-bottom: 1px solid grey;
+			text-align: center;
+		}
+		header nav a:first-child {
+			border-top: 1px solid grey;
+		}
+	}
```
Comment (PhilHart23) [inline css/index.css:129]: Either it's something on my end or inside of your code, but the nav isn't stacking on top of each other. Try on your end once more to see if it's working

File: index.html [hunk 2]
```diff
@@ -11,6 +13,85 @@
...
+
+    <footer>
+      <p>Copyright Great Idea! 2018</p>
+    </footer>
+
+  </div><!-- container -->
+
 </body>
+
 </html>
```
Comment (PhilHart23) [inline index.html:97]: The rest of this looks great!

**Comment by PhilHart23** [comment]
Great job on this project! Double check on the nav to make sure the media query works properly, it could very well be an issue on my end

File: css/index.css (hunk 1 above)
Comment (christopherc1331) [reply css/index.css:129]: Hey Phil,
So I don't know why it's doing this, but I can only seem to prompt the phone size media breakpoint to trigger if I have the dev tools open on my browser (see attached).  This doesn't surprise me, as I had the dev tools open the whole time I was working on this project and wouldn't have looked at my project without inspecting.  My media query breakpoint is set to trigger at 400px, but really doesn't unless I have dev tools open.  Do you have any idea why it's behaving like this?
Dev Tools Error

**Pull request merged**